### ISSUE TITLE
Rewording "Recent Podcasts"

### ISSUE THREAD

**Issue by silvernode** [Podcastindex-org/web-ui#89]
Scroll for TL;DR
I just thought this was at least worth discussing if it has not been already.
Thanks for the hard work and the feedback!

This is my first contribution to Podcasting 2.0, and while it's not a technical solution, it has to do with branding which one could argue is equally as important. 

Through the eyes of a newcomer to [podcastindex.org](https://podcastindex.org), one of the first things that is noticeable on the page is the "Recent Podcasts" widget (section?) which allows a person to browse and discover seemingly random episodes from podcasts contained inside the index. I found myself going from excitement to subtle confusion when I couldn't decide whether or not these episodes were actually randomly chosen or specifically featured by the admin of podcastindex. I leaned more toward thinking these episodes were chosen by the admin at first glance despite the fact that the word "featured" is not present but realized after refreshing the page that there is some sort of randomization going on.

At this point it hit me that a true newcomer might automatically assume that these podcasts were featured on purpose. This has the potential to reflect negatively on the project if the _wrong_ type of podcast shows up and is terrible in one way or another (either bad production or something generally offensive). 

If these podcasts are actually random, I suggest that the words "Recent Podcasts" be changed to "Pod Roulette" or "Random Podcasts". This way if anything shows up that isn't desired, the newcomer knows that it has nothing to do with the admin or the project as a whole. I think small changes like this go a long way to make the experience better and clearly understood by anyone who lands on the page for the first time.

TL;DR: "Recent Podcasts" on the official project website may make it appear there are specially featured podcasts instead of randomly selected podcasts. Change wording to better represent what is actually being shown such as "Pod Roulette", "Random Podcasts" or "Random Recent Podcasts".
